### DISCUSSION TITLE
Failure to Make, debian distro Parrot OS. "error: failed to parse manifest at `/opt/eruption/Cargo.toml`"

### DISCUSSION THREAD

**Discussion by wormbones** [X3n0m0rph59/eruption#209] [Q&A]
I'm trying to install but I seem to not be able to get past the make. It's holding up here.
I'm on 
Linux parrot 6.0.0-12parrot1-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.0.12-1parrot1 (2023-01-12) x86_64 GNU/Linux
I've followed the Debian I can, but I can't find anything about this at all. 
strip = false

`error: failed to parse manifest at `/opt/eruption/Cargo.toml`

Caused by:
  invalid type: unit variant, expected string only for key `profile.dev.strip`
make: *** [Makefile:31: build] Error 101`

`$cat /opt/eruption/Cargo.toml`

`[workspace]
members = ["eruption", "eruptionctl", "eruption-hwutil", "eruption-netfx", "eruption-debug-tool", "eruption-util",
    "eruption-hotplug-helper", "eruption-audio-proxy", "eruption-fx-proxy", "eruption-process-monitor",
    "pyroclasm", "eruption-gui-gtk3", "eruption-cmd", "eruption-keymap", "eruption-macro", "eruption-watchdog",
    "sdk/lib/rust/eruption-rs", "sdk/lib/rust/liberuption", "sdk/examples/rust/simple"]
default-members =  ["eruption", "eruptionctl", "eruption-hwutil", "eruption-netfx", "eruption-debug-tool", "eruption-util",
    "eruption-hotplug-helper", "eruption-audio-proxy", "eruption-fx-proxy", "eruption-process-monitor",
    "eruption-gui-gtk3", "eruption-cmd", "eruption-keymap", "eruption-macro",
    "sdk/lib/rust/eruption-rs", "sdk/lib/rust/liberuption"]
resolver = "2"

[profile.dev]
opt-level = 0
debug = true
rpath = false
lto = false
strip = false
debug-assertions = true
overflow-checks = true
codegen-units = 512
incremental = true
panic = "unwind"
\# split-debuginfo = "unpacked"

[profile.release]
opt-level = 3
debug = false
rpath = false
lto = true
strip = true
debug-assertions = false
overflow-checks = true
codegen-units = 1
incremental = false
panic = "abort"
\# split-debuginfo = "unpacked"

[profile.sccache]
inherits = "dev"
opt-level = 0
debug = true
rpath = false
lto = false
strip = false
debug-assertions = true
overflow-checks = true
codegen-units = 512
incremental = false
panic = "unwind"

[profile.profiling]
inherits = "release"
debug = 1`

Any help would be, of a great help. :)

**Comment by X3n0m0rph59** [comment]
Hi @not-mynumbness, looks like an outdated version of rust... Could you please try to download `rustup` from https://rustup.rs/ and install the latest stable rust compiler?

To build Eruption from source you require at least rust version `1.65`
